docs/getting-started-guides/ubuntu.md

```diff
@@ -56,6 +56,9 @@ work, which has been merge into this document.
 1. The nodes have installed docker version 1.2+ and bridge-utils to manipulate linux bridge.
 2. All machines can communicate with each other, no need to connect Internet (should use
 private docker registry in this case).
+
+    **Note:** When deploying, master need to connect Internet to download the necessary files.
+
```

Comment:
Prerequisites is a list (See also https://guides.github.com/features/mastering-markdown/#examples) while this is a sidenote. It would be much more readable to keep it as body.